src/openeo_grass_gis_driver/main.py

```diff
@@ -10,6 +10,7 @@
 __email__ = "soerengebbert@googlemail.com"
 
 
+create_endpoints()
+
 if __name__ == '__main__':
-    create_endpoints()
```

Comment:
Why are you moving `create_enpoints()` out?

Reply:
Now the more stable gunicorn wsgi server is used to run the openeo app. This way, `main.py` is not called directly anymore (before done via `python3 -m openeo_grass_gis_driver.main`) and `if __name__ == '__main__':` does not match anymore. This is the same mechanism [actinia_core](https://github.com/mundialis/actinia_core/blob/master/src/actinia_core/main.py) is using for stable production server while still be able to debug locally.